app/services/food_service.rb

```diff
@@ -0,0 +1,18 @@
+class FoodService
+  def self.return_food_query(food_input)
+    response  =  conn.get("fdc/v1/foods/search?query=#{food_input}")
+    parse_response(response)
+  end
+
+  private
+  def self.conn
+    conn = Faraday.new("https://api.nal.usda.gov") do |faraday|
+         faraday.params['api_key'] = ENV['food_token']
+       end
+  end
```

Comment:
You do not need to save this into a variable on line 9. This whole block is essentially the last line of your method, so this connection will get returned without declaring the variable.